test/e2e/apps/statefulset.go

```diff
@@ -510,6 +512,155 @@ var _ = SIGDescribe("StatefulSet", func() {
 
 		})
 
+		ginkgo.It("should perform canary updates and phased rolling updates of template modifications for partiton1 and delete pod-0", func(ctx context.Context) {
```

Comment:
```suggestion
		ginkgo.It("should perform canary updates and phased rolling updates of template modifications for partiton1 and delete pod0", func(ctx context.Context) {
```
nit: alternatively, use pod-0 throughout the test consistently.

Reply:
changed it to pod-0 throughout the test